src/jmclient/wallet.py

```diff
@@ -416,7 +416,7 @@ def max_mix_depth(self):
     def gaplimit(self):
         return self.gap_limit
 
-    def _load_storage(self):
+    def _load_storage(self, load_cache=True):
```

Comment:
```suggestion
    def _load_storage(self, load_cache: bool = True) -> None:
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1616/commits/ef1d76e9db09170db94ac0c10efe7b73866b64e3